general/database/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: '3.8'
+services:
+  mongo:
+    image: mongo:7
+    container_name: mongo
+    user: root
```

Comment:
Can we run service as a non-root user?

Reply:
Yes we can, I will remove this next push.